fs/fs.go

```diff
@@ -282,7 +259,7 @@ func (self *RealFsInfo) GetDirFsDevice(dir string) (*DeviceInfo, error) {
 }
 
 func (self *RealFsInfo) GetDirUsage(dir string) (uint64, error) {
-	out, err := exec.Command("du", "-s", dir).CombinedOutput()
+	out, err := exec.Command("/bin/sh", "-c", "ulimit -t 1; du -s", dir).CombinedOutput()
```

Comment:
Isn't it the larger docker containers that will take longer? Setting the cpu time this low will likely mean no fs stats for large containers, which reduces the usefulness of this check: finding which containers are using up your disk.